### DISCUSSION TITLE
Diffculty in installing ScatterJ in Fiji plugins

### DISCUSSION THREAD

**Discussion by uhuegbue** [community/community#141080] [New to GitHub]
Hello everyone, 
 I am having issues installing ScatterJ plugin in Fiji. Can someone help with this pleas?

**Comment by huseyin774** [comment]
To install the Scatter plugin in Fiji from GitHub, follow these steps:
Installation Steps
Download Fiji: If you haven't already installed Fiji, download it from the official ImageJ website. Ensure that Java is pre-installed on your computer (Windows, Mac OS, or Linux) .
Obtain the Plugin:
Navigate to the GitHub repository containing the Scatter plugin. You will need to download the JAR file for the Scatter plugin (the specific name may vary based on the repository) .
For example, if you are installing a dimensionality reduction plugin that includes scatter plotting capabilities, you would look for a file named something like Dimensionality_Reduction-1.0.x.jar .
Move the JAR File:
Copy the downloaded JAR file into the plugins subfolder of your Fiji installation directory. This can typically be found at:
Fiji.app/plugins/
Alternatively, you can drag and drop the JAR file directly into the main Fiji window .
Restart Fiji: After moving the JAR file, restart Fiji to ensure that it recognizes the newly installed plugin.
Accessing the Plugin: Once restarted, you can access the Scatter plugin from the Plugins menu in Fiji. The exact location within the menu may depend on how the plugin is structured 
Additional Notes
If you encounter any issues, ensure that you have all dependencies required by the plugin. Some plugins may require additional libraries to function correctly 
For specific usage instructions or features of the Scatter plugin, refer to any documentation provided in the GitHub repository or within Fiji itself once the plugin is installed.
By following these steps, you should be able to successfully install and use the Scatter plugin in Fiji.

good luck

**Comment by uhuegbue** [comment]
still could not find the file path

**Reply by huseyin774** [reply]
The general guidance of this matter is what I have stated
Do the steps one more time
good luck

**Comment by ghostinhershell** [comment]
Hi @uhuegbue,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as [the plugin's official website](https://savannah.nongnu.org/projects/scatterjn). It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
